.travis.yml

```diff
@@ -32,7 +32,7 @@ before_script:
 - nohup python -m httpbin.core --port 64526 &
 - while ! nc -q 1 127.0.0.1 64526 </dev/null; do sleep 2; done
 - curl http://127.0.0.1:64526/get?show_env=1
-- make
+- make -e
```

Comment:
The problem is that the behavior is already different per-platform, as the tools are different.  We use the same version of emscripten everywhere, but the native build uses native compilers.  The Travis build system's version of g++ we're using requires -std=c++11 for access to the complex trig functions we need, but other platforms don't appear to.  I did it this way because it was the easiest way to address a problem specific to the travis native build, but it could be done in the Makefile as well.